README.md

```diff
@@ -1,69 +1,61 @@
-# go-gh beta
+# Go library for the GitHub CLI
 
-**This project is in beta!** Feedback is welcome.
+`go-gh` is a collection of Go modules to make authoring [GitHub CLI extensions][extensions] easier.
 
-A Go module for CLI Go applications and [gh extensions][extensions] that want a convenient way to interact with [gh][], and the GitHub API using [gh][] environment configuration.
+Modules from this library will obey GitHub CLI conventions by default:
 
-`go-gh` supports multiple ways of getting access to `gh` functionality:
+- [`CurrentRepository()`](https://pkg.go.dev/github.com/cli/go-gh#CurrentRepository) respects `GH_REPO` and reads from git remote configuration as fallback;
+- GitHub API requests will be authenticated using the same mechanism as gh, respecting `GH_TOKEN` and `GH_HOST` if present;
+- Inspecting [`term`](https://pkg.go.dev/github.com/cli/go-gh/pkg/term) capabilities is done with respect to e.g. `NO_COLOR`;
+- Generating [table-](https://pkg.go.dev/github.com/cli/go-gh/pkg/tableprinter) or [Go template](https://pkg.go.dev/github.com/cli/go-gh/pkg/template)-based output uses the same engine as gh;
+- The [`browser`](https://pkg.go.dev/github.com/cli/go-gh/pkg/browser) module activates the user's preferred web browser.
 
-* Helpers that automatically read a `gh` config to authenticate themselves
-* `gh.Exec` shells out to a `gh` install on your machine
-
-If you'd like to use `go-gh` on systems without `gh` installed and configured, you can provide custom authentication details to the `go-gh` API helpers.
+## Usage
 
+See the [go-gh full reference documentation](https://pkg.go.dev/github.com/cli/go-gh).
```

Comment:
Nit:
```suggestion
See the full `go-gh`  [reference documentation](https://pkg.go.dev/github.com/cli/go-gh) for more information
```
also suggest putting this with "See [examples](https://github.com/cli/go-gh/blob/8457ec84b80a60d0e06a01158258a53af04293e4/example_gh_test.go) for more demonstrations of usage." after the example code